packages/components/src/local/molecules/sortable-list/types/props.d.ts

```diff
@@ -2,6 +2,8 @@ import { ReactNode, ReactText } from 'react'
 
 export interface ItemObject {
   name: string
+  /** If this role is `Game Control` - a role which cannot be deleted */
+  isGameControl?: boolean
```

Comment:
Ooh no, I don't like this @sanjayradadiya 

Sortable list is used in a range of places, including where `isGameControl` isn't present.

I know it's an optional prop - but I'd rather not see it here.

I don't know what the alternatives are, but I _guess_ `canDelete` callback could be provided that return true/false on if the row can be deleted.  It would popup the warning message if it receives a `gameControl` user, and return false.

Or, `maybe` this isn't needed, and we just access `isGameControl` from the item.

Reply:
I will recheck this and will find out way around to avoid this

Reply:
`isGameControl` removed from sortable list.

introduce new callback function for role delete with notification `Role EXCON with Game Control permissions cannot be deleted. Please remove Game Control permission.`